infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -333,6 +316,7 @@ public synchronized void trimCache(CorfuPayloadMsg<Long> msg,
                 entries++;
             }
         }
+        globalLogStart.set(trimMark);
```

Comment:
Probably need to rename TX_ABORT_NEWSEQ to TX_ABORT_NOINFO, or just add a new abort cause